target/linux/malta/config-5.4

```diff
@@ -33,15 +33,15 @@ CONFIG_BLK_DEV_RAM=y
 CONFIG_BLK_DEV_RAM_COUNT=16
 CONFIG_BLK_DEV_RAM_SIZE=4096
 CONFIG_BLK_DEV_SD=y
-CONFIG_BLK_DEV_SR=y
+# CONFIG_BLK_DEV_SR is not set
 # CONFIG_BLK_DEV_SR_VENDOR is not set
```

Comment:
With `kernel_menuconfig`, these lines should be dropped.  This will do when other people does this in future config refreshes.

fyi,

```
--- a/target/linux/malta/config-5.4
+++ b/target/linux/malta/config-5.4
@@ -33,15 +33,12 @@ CONFIG_BLK_DEV_RAM=y
 CONFIG_BLK_DEV_RAM_COUNT=16
 CONFIG_BLK_DEV_RAM_SIZE=4096
 CONFIG_BLK_DEV_SD=y
-CONFIG_BLK_DEV_SR=y
-# CONFIG_BLK_DEV_SR_VENDOR is not set
 CONFIG_BLK_MQ_PCI=y
 CONFIG_BLK_SCSI_REQUEST=y
 CONFIG_BOARD_SCACHE=y
 CONFIG_BOOT_ELF32=y
 CONFIG_BOUNCE=y
 CONFIG_BUILTIN_DTB=y
-CONFIG_CDROM=y
 CONFIG_CEVT_R4K=y
 # CONFIG_CGROUPS is not set
 CONFIG_CLKBLD_I8253=y
```

Reply:
Sure, I will do that. Could you explain the refresh behaviour and its problem with the previous change? That wasn't clear.